pyvistaqt/plotting.py

```diff
@@ -339,8 +343,20 @@ def __init__(self, parent=None, title=None, off_screen=None,
                  point_smoothing=False, polygon_smoothing=False,
                  splitting_position=None, auto_update=5.0, **kwargs):
         """Initialize Qt interactor."""
-        super(QtInteractor, self).__init__(parent=parent, **kwargs)
+        log.debug('QtInteractor init start')
+        # Cannot use super() here because
+        # QVTKRenderWindowInteractor silently swallows all kwargs
+        # because they use **kwargs in their constructor...
+        qvtk_kwargs = dict(parent=parent)
+        for key in ('stereo', 'iren', 'rw', 'wflags'):
+            if key in kwargs:
+                qvtk_kwargs[key] = kwargs.pop(key)
+        with _no_BasePlotter_init():
```

Comment:
It's still a hack, but I think it's cleaner than using `QVTKRenderWindowInteractorAdapter` and it's a move in the right direction.

Reply:
This is a solid case for monkey patching